src/Compilers/Core/Portable/Operations/IDelegateCreationExpression.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+namespace Microsoft.CodeAnalysis.Semantics
+{
+    /// <summary>
+    /// Represents a delegate creation expression. This is created whenever a new delegate is created.
+    /// </summary>
+    /// <remarks>
+    /// This interface is reserved for implementation by its associated APIs. We reserve the right to
+    /// change it in the future.
+    /// </remarks>
+    public interface IDelegateCreationExpression : IOperation
+    {
+        /// <summary>
+        /// The conversion and lambda body or method binding that this delegate is created from.
```

Comment:
>The conversion and lambda body [](start = 12, length = 30)

Is this accurate? Also, could this ever be an invalid expression?

Reply:
I updated the comment, as it wasn't accurate on the conversion scenario anymore. Looking at my tests so far, it does not appear that this will ever be an IInvalidExpression. The only NoneOperations we have the moment here are BoundQueryLambdas, which https://github.com/dotnet/roslyn/pull/21356 will address by turning them into IAnonymousFunctionExpressions.

Reply:
As a follow up, it appears that for truly invalid syntax, such as `new Action(gibberish)` or `new Action(Method1, Method2)`, this fails to bind entirely, and we return an `IInvalidExpression`. I'll add some tests for this.